cmd/vic-machine/common/dns.go

```diff
@@ -25,15 +25,15 @@ import (
 
 // general dns
 type DNS struct {
-	dns   cli.StringSlice `arg:"dns-server"`
+	Dns   cli.StringSlice `arg:"dns-server"`
```

Comment:
Should this be called `DNSServers` and use `arg:dns-servers`?

Reply:
Or maybe `NameServers`?

Reply:
just changed it to NameServers...go lint complained about the names